### PR TITLE
Release 2023 07 13 3

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Release candidate changes.

# Changes

- Changelog
- Patch versions

### CODE REVIEW

**Review by lmuntaner — APPROVED** [review]
Thanks!

File: CHANGELOG-Nns-Dapp.md [hunk 1]
```diff
@@ -11,6 +11,36 @@ The NNS Dapp is released through proposals in the Network Nervous System. Theref
...
+
+#### Security
+
+#### Not Published
+
+### Operations
+
+#### Added
+
+* Better visibility of upgrade cycle consumption, wasm size and memory usage.
```
Comment (lmuntaner) [inline CHANGELOG-Nns-Dapp.md:28]: Is this really the only changes after the RC was cut?

Comment (mstrasinskis) [reply CHANGELOG-Nns-Dapp.md:28]: I didn't find any other changes.

File: CHANGELOG-Nns-Dapp.md [hunk 2]
```diff
@@ -39,7 +69,7 @@ The NNS Dapp is released through proposals in the Network Nervous System. Theref
 
 #### Added
 
-* Better visibility of upgrade cycle consumption, wasm size and memory usage.
+* Better visibility of wasm size in CI.
```
Comment (dskloetd) [inline CHANGELOG-Nns-Dapp.md]: Should this really be both here and above on line 28? Are they different changes that happen both before and after the release?

File: CHANGELOG-Nns-Dapp.md [hunk 3]
```diff
@@ -11,6 +11,36 @@ The NNS Dapp is released through proposals in the Network Nervous System. Theref
...
+#### Fixed
+
+#### Security
+
+## Proposal 123471
+### Application
+
+#### Added
+
 * Nns voting e2e test.
```
Comment (dskloetd) [inline CHANGELOG-Nns-Dapp.md:44]: Tests are not part of the application so they should go in the Operations section.

File: CHANGELOG-Nns-Dapp.md [hunk 4]
```diff
@@ -11,6 +11,36 @@ The NNS Dapp is released through proposals in the Network Nervous System. Theref
...
 
 #### Changed
```
Comment (dskloetd) [inline CHANGELOG-Nns-Dapp.md:46]: Line 51 is just a wrong version of line 52. Should it be removed?

File: CHANGELOG-Nns-Dapp.md (hunk 1 above)
Comment (dskloetd) [reply CHANGELOG-Nns-Dapp.md:28]: You can verify here: https://github.com/dfinity/nns-dapp/blob/release-candidate/CHANGELOG-Nns-Dapp.md

File: CHANGELOG-Nns-Dapp.md (hunk 2 above)
Comment (mstrasinskis) [reply CHANGELOG-Nns-Dapp.md]: Thanks, the line was removed.

File: CHANGELOG-Nns-Dapp.md (hunk 3 above)
Comment (mstrasinskis) [reply CHANGELOG-Nns-Dapp.md:44]: Good point, was moved under the Operations.

File: CHANGELOG-Nns-Dapp.md (hunk 4 above)
Comment (mstrasinskis) [reply CHANGELOG-Nns-Dapp.md:42]: Good catch! Done.

**Review by dskloetd — APPROVED** [review]

**Pull request merged**